io.github.louis77.tuner.json

```diff
@@ -0,0 +1,111 @@
+    "runtime-version" : "24.08",
+
+    "sdk" : "org.freedesktop.Sdk",
+
+    "add-extensions" : {
+        "org.freedesktop.Sdk.Extension.vala" : {
+        "version" : "24.08",
+        "directory" : "vala",
+        "autodelete" : true
+        }
+    },
+
+    "build-options" : {
+        "prepend-path" : "/usr/lib/sdk/vala/bin/",
+        "prepend-ld-library-path" : "/usr/lib/sdk/vala/lib"
+        },    
+
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=xdg-desktop",
+        "--share=network",
+        "--metadata=X-DConf=migrate-path=/io/github/louis77/tuner/",
+        "--socket=pulseaudio",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.mpris.MediaPlayer2.Tuner",
+        "--env=PATH=/usr/bin:/app/bin:/app/vala/bin",
+        "--env=LD_LIBRARY_PATH=/app/vala/lib:/app/lib"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "vala",
+            "buildsystem" : "simple",
+            "build-commands" : ["install -d /app/vala"]
+        },
+        {
+            "name" : "libsoup",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+                "--buildtype=release",
+                "-Dvapi=enabled",
+                "-Dtests=false",
+                "-Dgssapi=disabled",
+                "-Dsysprof=disabled"
+            ],
+            "sources" : [{
+                "type" : "git",
+                "url" : "https://gitlab.gnome.org/GNOME/libsoup.git",
+                "tag" : "3.6.0",
+                "commit" : "3f03218b39306870af8b7331bd2e4c18ad4ce4d0"
+            }]
+        },
+        {
+            "name" : "gee",
+            "buildsystem" : "autotools",
+            "build-options" : {
+                "cflags" : "-Wno-error=incompatible-pointer-types"
+            },
+            "cleanup" : ["*"],
+            "sources" : [{
+                "type" : "archive",
+                "url" : "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz",
+                "sha256" : "1bf834f5e10d60cc6124d74ed3c1dd38da646787fbf7872220b8b4068e476d4d"
+            }]
+        },
+        {
+            "name" : "tuner",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release"
+            ],
+            "post-install" : [
+                "cp -n -v /app/share/icons/hicolor/16x16/actions/*  /app/share/icons/hicolor/scalable/actions/",
+                "install -Dm644 /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.svg  -t /app/share/icons/hicolor/scalable/apps/",
```

Comment:
this should be fixed upstream.

Reply:
This is a copy from the current flathub manifest - what needs to be fixed exactly?